packages/cli/src/lib/platform/authenticatedClient.spec.ts

```diff
@@ -156,7 +156,13 @@ describe('AuthenticatedClient', () => {
           description: 'Generated by the Coveo CLI',
           enabled: true,
           privileges: [
-            {targetDomain: 'IMPERSONATE', targetId: '*', owner: 'SEARCH_API'},
+            {owner: 'SEARCH_API', targetDomain: 'IMPERSONATE', targetId: '*'},
```

Comment:
I think you should add a second optional parameter `additionnalPrivileges` for `createImpersonateApiKey` that takes an array of privileges.

Because currently, the changes you're doing to the method is affecting every `ui:create` commands in the end, despite none of them other than the atomic one needing this right.